template_debian/distribution.sh

```diff
@@ -489,7 +489,7 @@ EOF
 deb [arch=amd64] https://deb.qubes-os.org/r${USE_QUBES_REPO_VERSION}/vm ${DIST_CODENAME}-testing main
 EOF
             fi
-        chroot_cmd apt-key add - < "${KEYS_DIR}/qubes-debian-r${USE_QUBES_REPO_VERSION}.asc"
+        cp "${KEYS_DIR}/qubes-debian-r${USE_QUBES_REPO_VERSION}.asc" "${INSTALL_DIR}/etc/apt/trusted.gpg.d/"
```

Comment:
When changing it to a separate file, then you can finally clean it up in `uninstallQubesRepo` function below. You may want to name it like qubes-builder.asc, to match the list file name and make cleanup easier (no separate cases for ubuntu).